deluge/ui/console/modes/input_popup.py

```diff
@@ -107,11 +107,10 @@ def __init__(self, parent, message, name, child, checked=False, additional_forma
         self.name = name
         self.checked = checked
         self.msglen = len(self.chkd_inact) + 1
-        self.child = child
```

Comment:
I'm not sure what fixed means here? This file's code changes should be in a separate branch and commits with explanation of the whole changeset.